tests/relay_integration/lang/java/test_plugin.py

```diff
@@ -1136,7 +1136,7 @@ def test_source_lookup_with_proguard(self):
             frames[24].module
             == "io.sentry.samples.instrumentation.ui.AnotherClassInSameFile$AnotherInnerClass"
         )
-        assert frames[24].lineno == 30
+        assert frames[24].lineno == 26
```

Comment:
the parsing changed here -- looks to be intentional though?

Reply:
Yes this is intentional. See https://github.com/getsentry/rust-proguard/pull/27 for more details. We're now aligning with Proguards `retrace.jar` output for callbacks. Just did the same fix here: https://github.com/getsentry/sentry/pull/51576

I'll go ahead and close the other PR in favor of this one.